lib/pacemaker/pcmk_cluster_queries.c

```diff
@@ -36,6 +36,7 @@ typedef struct {
     int rc;
     guint message_timer_id;
     guint message_timeout_ms;
+    void *private;
```

Comment:
Definitely can. The idea was that the `private` member would be simple to extend later to use for something else in other callbacks, whereas that particular enum likely wouldn't be. For example, the same `data_t` struct type is passed to `controller_status_event_cb()` and `designated_controller_event_cb()`.

Since this is internal as you mentioned, it also wouldn't add multiple new fields as needed, and either use them or ignore them as appropriate.